docs/releases/1.10.txt

```diff
@@ -544,6 +544,31 @@ This prevents confusion about an assignment resulting in an implicit save.
   :class:`~django.contrib.gis.geos.MultiPolygon` is deprecated in favor of the
   :attr:`~django.contrib.gis.geos.GEOSGeometry.unary_union` property.
 
+``CommaSeparatedIntegerField``
+----------------------------------------------------------
+
+``CommaSeparatedIntegerField`` has been deprecated in favor of 
+:class:`django.db.models.CharField`. To get the same behavior you will need to
+add the ``validate_comma_separated_integer_list`` validator to your field.
+
+    from django.core.validators import validate_comma_separated_integer_list
+    from django.db import models
+
+    class MyModel(models.Model):
+        csi = models.CharField(validators=[validate_comma_separated_integer_list])
+
```

Comment:
Not sure why the build is failing here. @timgraham, is there another way to do code blocks like this?

```
+ echo Spelling errors (please add real words to docs/spelling_wordlist):'
Spelling errors (please add real words to docs/spelling_wordlist):'
+ cat _build/spelling/output.txt
releases/1.10.txt:568: (csi) 
releases/1.10.txt:568: (CharField) 
releases/1.10.txt:580: (csi) 
releases/1.10.txt:580: (TextField) 
+ [ -s doc_warnings ]
+ exit 1
Build step 'Execute shell' marked build as failure
```